website/docs/guides/using-directives.md

```diff
@@ -12,25 +12,25 @@ The `using` directives mechanism lets you define configuration information withi
 `using` directives are basically key-value pairs that let you provide multiple values to a single key. For instance, this command:
 
 ```scala
-using foo bar baz
+using foo "bar", "baz"
 ```
 
 will be interpreted as assigning `bar` and `baz` to the key `foo`.
 
 As shown, `using` directives can be defined using the special keyword `using`. However, this may not be compatible with exisiting tools outside of Scala CLI. Therefore, they can be used in comments:
 
 ```scala
-// using scala 2
+// using scala "2"
```

Comment:
Do we need quotes around numbers? I was under impression that Strings and primitives are valid values

Reply:
scala version is usually a string so basically I assumed we accept String there. Numbers are also valid values but for scala version they are not retrieved from directive. It can be changed though